posts/MarkoB/2024-09-04-hibernate-validator-9-0-0-Beta3.adoc

```diff
@@ -0,0 +1,105 @@
+= Hibernate Validator 9.0.0.Beta3 is out
+Marko Bekhta
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+:hibernate-validator-doc-url-prefix: https://docs.jboss.org/hibernate/validator/9.0/reference/en-US/html_single/
+:hibernate-validator-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hibernate-validator-version-family: 9.0
+:hibernate-validator-jira-project-id: 10060
+:hibernate-validator-jira-version-id: 32328
+---
+
+We just published Hibernate Validator 9.0.0.Beta3,
+the next beta release of the new 9.0 series of Hibernate Validator.
+
+This series targets Jakarta EE 11. It is the implementation of the Jakarta Validation 3.1.
+Compared to the previous release in this series this version
+removes a few more constraints, configuration properties and APIs
+that have been deprecated for a few major versions now.
+We have also identified a few possible areas to improve while testing
+the previous version in the downstream projects.
+Also, there are a few fixes.
```

Comment:
You used "a few" four times in three sentences :)

```suggestion
Compared to the previous release in this series this version
removes a few more constraints, configuration properties and APIs
which have been deprecated for several major versions.
We have also identified some possible areas to improve while testing
the previous version in downstream projects.
Also, there are some bugfixes.
```

Reply:
just ... a few times 🙈🙈🙈🙈

Thanks for the suggestion!!! 😃